app/controllers/users/sessions_controller.rb

```diff
@@ -116,10 +116,6 @@ def handle_valid_authentication
       redirect_to next_url_after_valid_authentication
     end
 
-    def browser_is_ie11?
```

Comment:
I think we need to update the `@browser_is_ie11 = browser_is_ie11` inside `#new` above too?

Reply:
Ope, yep, removed